UniWatch/Controllers/ClassController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+
+namespace UniWatch.Controllers
+{
+    public class ClassController : Controller
+    {
+        private IClassManager _classManager;
+
+        public ClassController()
+        {
+            _classManager = new ClassManager();
+        }
+
+        //GET: Index
+        public ActionResult Index()
```

Comment:
I tried executing your code josh. I saw your view for index. I am seeing an error on your index action something to do with for loop

![image](https://cloud.githubusercontent.com/assets/14304976/20617455/0c4c7baa-b2af-11e6-8f14-4f390e8d45b2.png)